src/registry/mod.rs

```diff
@@ -423,7 +423,7 @@ pub struct Registry {
     pub subscription_type: Option<String>,
     pub introspection_mode: IntrospectionMode,
     pub enable_federation: bool,
-    pub suppress_apollo_link: bool,
+    pub enable_apollo_link: bool,
```

Comment:
Just wondering, instead of `enable_apollo_link` maybe better name would be `enable_federation_v2` or something along those lines. Outside of the library maintainers I am unsure whether devs would be aware of the significance here.